storybooknative/src/CenterView/index.js

```diff
@@ -4,13 +4,16 @@ import * as React from "react";
 import { View } from "react-native";
 
 import styled from "../styled";
+import type { StylePropType } from "../types/StylesPropType";
 
-const Main = styled(View, {
+const style: StylePropType = {
```

Comment:
if we need to do this, we should export StylePropType from same file as styled, since they will often be used together